plugin/storage/es/spanstore/reader_test.go

```diff
@@ -1267,7 +1252,7 @@ func TestSpanReader_GetEmptyIndex(t *testing.T) {
 }
 
 func TestSpanReader_ArchiveTraces(t *testing.T) {
-	withArchiveSpanReader(t, false, func(r *spanReaderTest) {
+	withArchiveSpanReader(t, true, "archive", func(r *spanReaderTest) {
```

Comment:
by passing the suffix explicitly (which no production code does today) you're overriding the default behavior. It's ok to do this as one of the tests, but we should keep the "default" tests where only `use_aliases` flag is varied and check that it affects the index name. And then for passing a suffix explicitly like there I would use a completely different string like `foobar`, to ensure that the validation is not mistaking the index name from default behavior with the index name with overwritten suffix.

Perhaps it makes sense to combine into a table driven test

| use_aliases | input suffix | expected index |
| ----------- | ------------ | -------------- |
| false | "" | ... |
| true | "" | ... |
| false | "foobar" | ... |
| true | "foobar" | ... |

Reply:
@yurishkuro Done. Can we remove the test below this now? I think